src/modules/Builder/pages/BuilderApplet/BuilderApplet.schema.ts

```diff
@@ -2,24 +2,64 @@ import * as yup from 'yup';
 
 import i18n from 'i18n';
 import { getMaxLengthValidationError } from 'shared/utils';
-import { MAX_DESCRIPTION_LENGTH, MAX_NAME_LENGTH } from 'shared/consts';
+import {
+  ItemResponseType,
+  MAX_DESCRIPTION_LENGTH,
+  MAX_NAME_LENGTH,
+  MAX_SELECT_OPTION_TEXT_LENGTH,
+  MAX_SLIDER_LABEL_TEXT_LENGTH,
+} from 'shared/consts';
 
 const { t } = i18n;
 
 export const getIsRequiredValidateMessage = (field: string) =>
   t('validationMessages.isRequired', { field: t(field) });
 
-export const ItemSchema = () =>
-  yup
+export const ResponseValuesRowsSchema = () => ({
+  minLabel: yup.string().max(MAX_SLIDER_LABEL_TEXT_LENGTH, getMaxLengthValidationError),
```

Comment:
could you please check if validation error have to be the same as for maxLabel?

Reply:
re-checked, it has to be the same